### PR TITLE
TypeError bug

### PR DESCRIPTION
**Pull request by berkorbay**
A quick fix to TypeError bug mentioned in #90 . It is a simple try-catch mechanism but the real issue might be deeper.

### CODE REVIEW

File: hanging_threads.py [hunk 1]
```diff
@@ -169,7 +169,13 @@ def frame2string(frame):
     filename = co.co_filename
     name = co.co_name
     s = '\tFile "{0}", line {1}, in {2}'.format(filename, lineno, name)
-    line = linecache.getline(filename, lineno, frame.f_globals).lstrip()
+    try:
+        line = linecache.getline(filename, lineno, frame.f_globals).lstrip()
```
Comment (niccokunzmann) [inline hanging_threads.py:173]: I would assume this is due to `lineno == None` which can be tested before the call of the function.

**Comment by niccokunzmann** [comment]
Thanks!

**Comment by niccokunzmann** [comment]
I left a little comment and if you or someone else likes, they can submit a different fix.

File: hanging_threads.py (hunk 1 above)
Comment (berkorbay) [reply hanging_threads.py:173]: I think so; but there was one other component that I was not very sure of. Therefore I thought try-catch would be a better patch here.

**Comment by niccokunzmann** [comment]
Thanks, this included in v2.0.7!

**Pull request merged**